### PR TITLE
Added: Files to fulfill the community standards.

### PR DESCRIPTION
**Pull request by Sven-Seyfert**
Hi @Danp2 , hi @mlipok ,

this pull request provides additional files to fulfill recommended community standards. It's all about contribution and maintenance on GitHub. That's why GitHub recommend such files to regulate the community behavior and to ensure an effective way of contributing to the project(s).

Further explanation [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/about-community-profiles-for-public-repositories) 😀 .

### Overview

- bug_report.md and feature_request.md are suggested files for adding or opening an issue
  - I adjusted the content just a little bit, the rest was generated by GitHub
- PULL_REQUEST_TEMPLATE.md helps to categorize
  - I adjusted the content just a little bit, the rest was generated by GitHub
- CODE_OF_CONDUCT.md defines standards for how to engage in a community, see [here](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types)
  - if we want to have such things, there should be a contact person or better a contact person email address that has to be set instead of the placeholder "email@address.com"

### Hints

I am aware that these things are not the focus of expanding the project or the the UDF, but it's even more professional and user friendly in my honest opinion. As far as I can see until now, you both are really focused and aware of improving the code, of expanding the functionality and ensure the W3C standards which is essential to the whole project, no question 👍 .
I want to ensure that this awesome project is presented and user friendly as best as possible 😇 .

- In case you want to close the PR with: "No, not needed [...]" or "It's not my/our approach to handle the project [...]" or similar, I am totally fine.
- In case you think it doesn't matter much, I suggest to do it => better standing in the GitHub community, better searchability, more recommendations and more.

> But as always, it's up to you 😁 .

Best regards
Sven

### CODE REVIEW

**Comment by mlipok** [comment]
please take a look here:
`https://github.com/osTicket/osTicket/issues/new`

They have som kind of form which automaticlly is created when you open new issue.

Are you able to do the same ?

**Comment by mlipok** [comment]
Or even better take a look here:
`https://github.com/BookStackApp/BookStack/issues/new/choose`

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,12 +17,15 @@ Go to [legend](#legend---types-of-changes) for further information about the typ
 	- bug_report.md and feature_request.md
 	- CODE_OF_CONDUCT.md file
 	- PULL_REQUEST_TEMPLATE.md file
+- _WD_JsonCookie() (mLipok)
 
 ### Changed
 
 - README.md restruturing
```
Comment (mlipok) [inline CHANGELOG.md:24]: restructuring ? typo?

Comment (Danp2) [reply CHANGELOG.md:24]: Must have been from when I "fixed" the conflicts. I will correct it.

Comment (Danp2) [reply CHANGELOG.md:24]: OIC.... You meant the parentheses. 🙄 I'll fix it.

Comment (mlipok) [reply CHANGELOG.md:24]: no parentheses.

`- README.md restruturing` >> `- README.md restructuring` there is missing `c`

Comment (Danp2) [reply CHANGELOG.md:24]: > restruturing 

Ah... ok. Wasn't clear what you meant. I saw that line 20 was highlighted and focused only on that.

Comment (mlipok) [reply CHANGELOG.md:24]: Line 20 was changed and for this reason is marked on green.
It is not focused line.
My review was focused on line 24, and this is something you have to get used to on github.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,12 +13,17 @@ Go to [legend](#legend---types-of-changes) for further information about the typ
...
 
+- Files to fulfill the community standards
+	- bug_report.md and feature_request.md
+	- CODE_OF_CONDUCT.md file
+	- PULL_REQUEST_TEMPLATE.md file
 - _WD_JsonCookie() (mLipok)
 
 ### Changed
 
 - README.md restruturing
```
Comment (Danp2) [inline CHANGELOG.md:24]: > My review was focused on line 24, and this is something you have to get used to on github.
btw. I only noticed it myself last week.

If you click on the row number of the target row before clicking the plus sign, then it should highlight the correct row for the reviewer. This is a test of that functionality.

Comment (Danp2) [reply CHANGELOG.md:24]: Hmmm... I guess it doesn't work the way I thought. ☹️

**Pull request merged**